analyzers/tests/SonarAnalyzer.UnitTest/TestFramework/TestHelper.cs

```diff
@@ -185,6 +185,13 @@ public static IMethodSymbol GetMethodSymbol(this (SyntaxTree, SemanticModel) tup
             return semanticModel.GetDeclaredSymbol(syntaxTree.GetMethod(name, skip));
         }
 
+        public static IFieldSymbol GetFieldSymbol(string fieldDefinition)
+        {
+            var compiler = new SnippetCompiler($@"class __C__ {{ {fieldDefinition} }}");
```

Comment:
The `__C__` raises questions about why is it called like that and what does the special name means.
```suggestion
            var compiler = new SnippetCompiler($@"class Sample {{ {fieldDefinition} }}");
```

Reply:
I named it "C" which is less likely to collide with "Sample".